packages/template-block/src/components/AlertError.tsx

```diff
@@ -0,0 +1,18 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { IconExclamationMarkTriangle } from '@frontify/fondue';
+
+export type AlertErrorProps = {
+    errorMessage: string;
+};
+
+export const AlertError = ({ errorMessage }: AlertErrorProps) => {
+    return (
+        <span className="tw-p-5 tw-flex tw-gap-4 tw-mb-4 tw-items-center tw-rounded-sm tw-transition-opacity tw-ease-in-out tw-bg-box-negative tw-text-box-negative-inverse">
```

Comment:
Maybe for better accessibility add `aria-live="assertive"` so it gets announce, basically the same https://github.com/Frontify/guideline-blocks/pull/676#discussion_r1356688686 in the other PR, maybe @fulopdaniel and you can ask @LidijaStoilova together if its needed or not

Reply:
Nice catch!
Lidija is on vacation until October 21st, so we have to decide on our own 🙂
I've set the `assertive` for now and saved the question for a later (when Lidija will back from vacation).

Reply:
`aria-live="assertive"` should be used for time-sensitive/critical notifications that require the user's immediate attention. If that is the case, you can keep it, otherwise you can use `aria-live="polite"`. Check [this link](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions) for more information.